CHANGELOG.md

```diff
@@ -8,6 +8,13 @@
 * Four permissions roles are supported and enforced: guest, contributor, editor and admin. See the documentation for details. Pre-existing alpha users are automatically migrated to the admin role, as they already could do anything.
 * The admin bar menu is fully responsive to user roles.
 * The context bar entirely appears or disappears based on user roles.
+* Contributors can submit drafts for review ("Propose Changes" versus "Publish Changes").
```

Comment:
Are these unrelated lines in here from some inadvertent branching?

Reply:
What unrelated lines? I only see one "+" here

Reply:
They are all accurate to what is currently in 3.0 (apart from the new one)